db/migrate/20220716112940_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :comment_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Nice job creating tables however this table should be associated with the user table. Kindly use a migration like this :point_down: 

```.rb
class CreatePosts < ActiveRecord::Migration[7.0]
  def change
    create_table :posts do |t|
      t.string :title
      t.string :text
      t.integer :comment_counter
      t.integer :likes_counter
      t.references :author, foreign_key: { to_table: 'users' }

      t.timestamps
    end
  end
end
```

Reply:
This issue is well resolved! ✔️